### PR TITLE
[documentation] runner - doc fix for env list of values

### PR DESCRIPTION
**Pull request by DariuszPorowski**
Fixes https://github.com/oxsecurity/megalinter/issues/2448 based on solution from https://github.com/oxsecurity/megalinter/issues/1659

## Proposed Changes

`'ENABLE=MARKDOWN,YAML'` -> `"'ENABLE=MARKDOWN,YAML'"`

### Author/Contributor
- [x] Add entry to the [CHANGELOG](https://github.com/oxsecurity/megalinter/blob/main/CHANGELOG.md) listing the change and linking to the corresponding issue (if appropriate)
- [x] If documentation is needed for this change, has that been included in this pull request

### Reviewing Maintainer
- [ ] Label as `breaking` if this is a large fundamental change
- [ ] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`

### CODE REVIEW

File: docs/mega-linter-runner.md [hunk 1]
```diff
@@ -44,7 +45,7 @@ You can run mega-linter-runner without installation by using `npx`
 Example:
 
 ```shell
-npx mega-linter-runner -r beta -e 'ENABLE=MARKDOWN,YAML' -e 'SHOW_ELAPSED_TIME=true'
+npx mega-linter-runner -r beta -e "'ENABLE=MARKDOWN,YAML'" -e 'SHOW_ELAPSED_TIME=true'
```
Comment (nvuillam) [inline docs/mega-linter-runner.md:48]: This files is generated from megalinter-runner/README.md , please can you update it ? :)

**Review by nvuillam — CHANGES_REQUESTED** [review]
Thanks for the PR :)
Please can you check the comment ?

File: docs/mega-linter-runner.md (hunk 1 above)
Comment (DariuszPorowski) [reply docs/mega-linter-runner.md:48]: @nvuillam done :)

**Review by nvuillam — APPROVED** [review]
Thanks a lot for the PR :)

**Pull request merged**